streaming/index.js

```diff
@@ -13,7 +13,6 @@ const { JSDOM } = require('jsdom');
 const log = require('npmlog');
 const pg = require('pg');
 const dbUrlToConfig = require('pg-connection-string').parse;
-const uuid = require('uuid');
```

Comment:
This is just dropped in this pull request, not in #27828, but linting was complaining.